identity/conf/routes

```diff
@@ -15,6 +15,7 @@ GET         /user/:vanityUrl                        controllers.PublicProfileCon
 GET         /user/:vanityUrl/:activityType          controllers.PublicProfileController.renderProfileFromVanityUrl(vanityUrl: String, activityType: String)
 
 GET         /complete-registration                  controllers.EmailVerificationController.completeRegistration
+GET         /resend-validation                      controllers.EmailVerificationController.resendValidationEmail
```

Comment:
we'd usually have an HTML form POSTing to a `/resend-validation` to trigger the resend and return the page.
It should be possible to create a very small form for posting to this endpoint, without changing  the design.